webapp/app.py

```diff
@@ -25,6 +26,17 @@ def index():
     return render_template("index.html")
 
 
+@app.route("/download/<osname>")
+def osredirect(osname):
+    if "macos" not in osname and "windows" not in osname:
+        return render_template("index.html")
+
+    SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
+    json_url = os.path.join(SITE_ROOT, "../static/", "latest-release.json")
```

Comment:
Either split `../static` or put together with the file name?

Reply:
fixed